plugins/auth-backend-module-vmware-cloud-provider/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-auth-backend-module-vmware-cloud-provider",
+  "description": "The vmware-cloud-provider backend module for the auth plugin.",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "workspace:^",
+    "@backstage/backend-plugin-api": "workspace:^",
+    "@backstage/catalog-model": "workspace:^",
+    "@backstage/errors": "workspace:^",
+    "@backstage/plugin-auth-node": "workspace:^",
+    "jose": "^4.6.0",
+    "passport-oauth2": "^1.6.1"
+  },
+  "devDependencies": {
```

Comment:
I think [these](https://github.com/backstage/backstage/actions/runs/6935447612/job/18865631857?pr=21337) [two](https://github.com/backstage/backstage/actions/runs/6935447612/job/18865632292?pr=21337) CI check failures want to see a `@types/passport-oauth2` entry here?